arches_lingo/pkg/graphs/resource_models/Concept.json

```diff
@@ -51,10 +67,18 @@
                     "description": "{\"en\": \"\"}",
```

Comment:
The description field on CardModel is I18n_TextField, so this is acceptable. It's the description field on the Node model that is plain TextField that should not have descriptions like `"{\"en\": \"\"}"`... (unless we decide that node descriptions also need to be internationalized and change that config).

I purged the faulty node descriptions (artifact from previously fixed bug) in 6c3477d79dbec60968380ad8bf450cd92f93cc60